assets/assets.json

```diff
@@ -3985,5 +3687,303 @@
   },
   "10b38465-640c-509a-b047-8d9671a245ae": {
     "coinGeckoId": "vechain-old-erc20"
+  },
```

Comment:
Do you know why a bunch of stuff is moved to the bottom here?

Reply:
Nope! No idea.